cmd/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"ldcli/cmd/cliflags"
+	"ldcli/internal/analytics"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type CommandRunEventType struct {
```

Comment:
I'm not sure we're getting much from this type instead of passing in the event name directly to `SendEvents`.

Reply:
I wanted to encapsulate the "buildProperties" for each different event type. The other option is to have multiple functions along the lines of BuildPropertiesForCommandRun(), BuildPropertiesForCommandCompleted().
Which I'm open to as well, just that the function name looked quite long.